src/Compilers/CSharp/Test/Semantic/Semantics/PatternMatchingTests4.cs

```diff
@@ -2052,7 +2052,7 @@ public class SwitchExpressionException : InvalidOperationException
                 //             _ = (1, 2) switch { (3, 4) => 1 };
                 Diagnostic(ErrorCode.WRN_SwitchExpressionNotExhaustive, "switch").WithLocation(8, 24)
                 );
-            CompileAndVerify(compilation, expectedOutput: "SwitchExpressionException()");
+            CompileAndVerify(compilation, expectedOutput: "SwitchExpressionException((1, 2))");
```

Comment:
Also, the meeting notes from [November](https://github.com/dotnet/csharplang/blob/4811c98606902cc85822e24c5dc5565a2bdadcab/meetings/2018/LDM-2018-11-14.md) list the exception as being called `MatchFailureException`. Did I miss a rename?

Reply:
Yes; see [https://github.com/dotnet/roslyn/issues/31911](https://github.com/dotnet/roslyn/issues/31911)

---
In reply to: [249960714](https://github.com/dotnet/roslyn/pull/32588#discussion_r249960714) [](ancestors = 249960714)